hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/MasterQuotaManager.java

```diff
@@ -740,5 +740,14 @@ public void processFileArchivals(FileArchiveNotificationRequest request, Connect
       notifier.addArchivedFiles(filesWithSize);
     }
   }
+
+  /**
+   * Remove regionInfo entry for the table which is going to get dropped.
+   *
+   * @param tableName tableName.
+   */
+  public void removeTableRegions(TableName tableName) {
+    regionSizes.entrySet().removeIf(regionInfo -> regionInfo.getKey().getTable().equals(tableName));
```

Comment:
nit: `entry` instead of `regionInfo`, please.

Reply:
Actually, looks like you could also do `regionSizes.keySet().removeIf(regionInfo -> regionInfo.getTable().equals(tableName))`.

Reply:
done.